build_packages

```diff
@@ -276,7 +276,7 @@ if [[ "${FLAGS_usepkgonly}" -eq "${FLAGS_FALSE}" ]]; then
   break_dep_loop sys-apps/util-linux udev,systemd,cryptsetup \
                  sys-fs/cryptsetup udev \
                  sys-fs/lvm2 udev,systemd \
-                 sys-apps/systemd cryptsetup,curl \
+                 sys-apps/systemd cryptsetup \
```

Comment:
Instead of dropping curl, please add importd. That something I did in the build tag, but forgot to put it into the PR branch (https://github.com/flatcar/scripts/commit/6e20bf308f56aaad24487e7de76491d37ce66dc7)

Reply:
As discussed on Matrix, there's no difference in overall packages built...